src/app/docs/components/search/search.tsx

```
@@ -0,0 +1,138 @@
+  if (!data) {
+    return null;
+  }
+
+  return (
+    <div>
+      <Link href={data.url}>
+        <h3>{data.meta.title}</h3>
+        <p dangerouslySetInnerHTML={{ __html: data.excerpt }} />
+      </Link>
+
+      {data.sub_results && data.sub_results.length > 0 && (
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)